setup.py

```diff
@@ -1,13 +1,37 @@
+import codecs
+import os
 from setuptools import find_packages, setup
 
+def read(*parts):
+    """Reads a file from a sub directory.
+
+    :param parts: components of the path to file
+    :return: the file contents
+    """
+
+    # intentionally *not* adding an encoding option to open, See:
+    #   https://github.com/pypa/virtualenv/issues/201#issuecomment-3145690
+    here = os.path.abspath(os.path.dirname(__file__))
+    return codecs.open(os.path.join(here, *parts), 'r').read()
```

Comment:
on a vraiment besoin de ça?
open.read() pose un problème avec notre README.rst?

Reply:
You're perfectly right, today readme.md is an ASCII file, so we don't need to complicate things.